terragrunt-run-all/action.yml

```diff
@@ -0,0 +1,30 @@
+name: Terragrunt Run All
+description: Terragrunt run-all
+inputs:
+  folder:
+    description: The target folder
+    required: true
+  iam-role:
+    description: The iam role to assume
+    required: true
+  terragrunt-bin-path:
+    description: The terragrunt bin path
+    required: false
+    default: ./terragrunt
+  run-all-action:
+    description: The terragrunt command args
+    required: true
+  additional-options-args:
+    description: Additional terragrunt option args
+    required: false
+
+runs:
+  using: composite
+  steps:
+        
+    - name: Terrgrunt Apply
+      shell: bash
+      env:
+        TERRAGRUNT_IAM_ROLE: ${{ inputs.iam-role }}
+      run: |
+        ${{ inputs.terragrunt-bin-path }} run-all ${{ inputs.run-all-action }} --terragrunt-non-interactive --terragrunt-working-dir ${{ inputs.folder }} ${{ inputs.additional-options-args }}
```

Comment:
If this action is wrapping a single `run` step, does it still make sense to have it as a separate action? Is it likely that this action will do other things before the `run-all`?

Reply:
Do you meant there would be some additional parameters between ${{ inputs.terragrunt-bin-path }} and "run-all"? Or that there might be an additional step before "terragrunt run-all"? If it's the latter, my thinking with this action is only used to run the terragrunt plan command, and abstract away the nuances for doing that

Reply:
The more I think about it, terragrunt run-all commands can get too nuanced to handle with an overly simple action such as this, so I think I'm going to remove it from this pr.